geopy/geocoders/googleplaces.py

```diff
@@ -0,0 +1,217 @@
+            to 'maps.google.co.uk' to properly bias results.
+
+        :param string scheme: Use 'https' or 'http' as the API URL's scheme.
+            Default is https. Note that SSL connections' certificates are not
+            verified.
+
+        :param dict proxies: If specified, routes this geocoder's requests
+            through the specified proxy. E.g., {"https": "192.0.2.0"}. For
+            more information, see documentation on
+            :class:`urllib2.ProxyHandler`.
+        """
+        super(GooglePlaces, self).__init__(
+            scheme=scheme, timeout=timeout, proxies=proxies, user_agent=user_agent
+        )
+
+        self.api_key = api_key
+        self.domain = domain.strip('/')
+        self.scheme = scheme
+        self.doc = {}
+
+        self.autocomplete_api = '%s://%s/maps/api/place/autocomplete/json' % (self.scheme, self.domain)
+        self.details_api = '%s://%s/maps/api/place/details/json' % (self.scheme, self.domain)
+
+    @staticmethod
+    def _format_components_param(components):
+        """
+        Format the components dict to something Google understands.
+        """
+        return "|".join(
+            (":".join(item)
+             for item in components.items()
+             )
+        )
+
+    def geocode(
+            self,
+            query,
+            exactly_one=False,
+            timeout=None,
+            offset=None,
+            location=None,
+            radius=None,
+            language=None,
+            types=None,
+            components=None,
+    ):  # pylint: disable=W0221,R0913
+        """
+        Geocode a location query.
+
+        :param string query: The address or query you wish to geocode.
+
+        :param bool exactly_one: Return one result or a list of results, if
+            available.
+
+        :param int timeout: Time, in seconds, to wait for the geocoding service
+            to respond before raising a :class:`geopy.exc.GeocoderTimedOut`
+            exception. Set this only if you wish to override, on this call
+            only, the value set during the geocoder's initialization.
+
+            .. versionadded:: 1.12
+
+        :param offset: The position, in the input term, of the last character that the service uses to match predictions.
+            For example, if the input is 'Google' and the offset is 3, the service will match on 'Goo'.
+            If no offset is supplied, the service will use the whole term.
+        :type offset: int
+
+        :type location: :class:`geopy.point.Point`, list or tuple of (latitude,
+            longitude), or string as "%(latitude)s, %(longitude)s"
+
+        :param int radius:  The distance (in meters) within which to return place results.
+            Setting a radius biases results to the indicated area, but may not fully restrict results to the specified area.
+
+        :param string language: The language in which to return results.
```

Comment:
Adding a link according to google places documentation: https://developers.google.com/maps/faq#languagesupport